### ISSUE TITLE
analyze series

### ISSUE THREAD

**Issue by yanntm** [numworks/epsilon#219]
Could we imagine another tab for the series "suites" in my fr version, besides graphing and table, that would recognize geometric and arithmetic series, 
and provide the expression u(n) as a function of n for series defined using a recursion ?

i.e. test if u(n+1)/u(n) or u(n+1)-u(n) resolve as constants and display the results of the test in an exploitable way

Note that other calculators do not have the feature afaik, but looking at a terminale S math book, it would be very useful (just to check the results done by hand ofc !)

Convergence and limit n->+inf would be nice too.

**Comment by AlainBusser** [comment]
This uses algebra and is not easy to implement on a calculator without much algebra, but if you define the sequence as _explicit_, it should be possible to modify the app so that "goto" allows one to choose Infinity, especially if the sequence  has been defined as "explicit". Then

- either the answer would be "Nan" which means that the sequence is not arithmetic neither geometric, or has been defined as recurrent instead of explicit,
- or the answer would be the limit, infinity or 0 

In the meantime, you can see the effect near the end of [this article](http://irem.univ-reunion.fr/spip.php?article698)  (written in CoffeeScript but it is possible also with Python, using `float("inf")` to choose the infinity as a number). This works only with explicit sequences but it is defined by the IEEE754 convention, so it should work with any binary machine.


If there be anything to add to the sequences app, I would advocate the look for the first index where the sequence goes over or under a predefined value, as this is useful on any baccalauréat subject, in any section.

To sum up :-1: for the automatic recognising of arithmetic or geometric sequences (maybe too heavy and probably less necessary than the threshold search) and :+1: for the display of the limit when the sequence is explicit (mainly because I hope to be able to do that :stuck_out_tongue_winking_eye: )

**Comment by parisseb** [comment]
The HP Prime calculator has a rsolve command that can solve arithmetic and arithmetico-geometric sequences.
